templates/zerver/subscriptions.html

```diff
@@ -3,7 +3,10 @@
 
     <div class="subscriptions">
       <h1><i class="icon-vector-exchange streams-icon"></i>{% trans "Streams" %}</h1>
-      <div class="alert" id="subscriptions-status"></div>
+	    <div class="alert" id="subscriptions-status">
+	      <span id="response"></span>
+	      <span id="close-subscriptions-status"><i class="icon-vector-remove"></i></span>
```

Comment:
This block seems indented more than it should be -- I think the `subscription-status` div should be aligned with the <h1>